### PR TITLE
feat: ws status API to return info about activation status

### LINKED ISSUE #1239: GET /projects/:id/events/status to return info about activation
As a graph API user, I'd like webhook-service's `GET /projects/:id/events/status` API to return info about activation status.

Acceptance criteria:

- [x] webhook-service's `GET /projects/:id/events/status`  to return an additional field informing about the project activation status:
```json
{
  "activated": true,
  "progress": {
    "done":       2,
    "total":      5,
    "percentage": 100.00
  },
  "details": {
    "status":  "in-progress|success|failure",
    "message": "transformation non recoverable failure"
  }
}
```
  - [ ] ~Project's path needs to be found by calling GL's `single project` API~
  - [ ] ~the path to be used while calling `GET /knowledge-graph/projects/:namespace/:name`; for `200` response code, `"activated": true`;  otherwise -> `"activated": false` should be returned.~

### PR DESCRIPTION
**Pull request by jachro**
closes #1239 

/deploy #persist

### CODE REVIEW

File: graph-commons/src/main/scala/io/renku/graph/model/events.scala [hunk 1]
```diff
@@ -248,6 +249,59 @@ object events {
...
+    lazy val stage:      EventStatusProgress.Stage      = EventStatusProgress.Stage(status)
+    lazy val completion: EventStatusProgress.Completion = EventStatusProgress.Completion(stage)
+  }
+
+  object EventStatusProgress {
+    import EventStatus._
+
+    trait Stage extends IntTinyType with Product with Serializable
+    object Stage {
+      def apply(s: EventStatus): Stage = s match {
```
Comment (eikek) [inline graph-commons/src/main/scala/io/renku/graph/model/events.scala:262]: That's nice!

File: webhook-service/src/main/scala/io/renku/webhookservice/eventstatus/StatusInfo.scala [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    }"""
+  }
+}
+
+private sealed trait Progress extends Product with Serializable
+
+private object Progress {
+
+  final case object Zero extends Progress {
+    lazy val finalStage: EventStatusProgress.Stage = EventStatusProgress.Stage.Final
```
Comment (eikek) [inline webhook-service/src/main/scala/io/renku/webhookservice/eventstatus/StatusInfo.scala]: I probably just don't see it in the PR, I was wondering why this field to a constant is necessary? Same in `NonZero`. It's not in the interface and always points to the same constant value.

**Review by eikek — APPROVED** [review]
👍🏼

File: webhook-service/src/main/scala/io/renku/webhookservice/eventstatus/StatusInfo.scala (hunk 2 above)
Comment (jachro) [reply webhook-service/src/main/scala/io/renku/webhookservice/eventstatus/StatusInfo.scala]: Yes, good point

**Pull request merged**